test/html-to-react-tests.js

```diff
@@ -101,6 +101,25 @@ describe('Html2React', function() {
 
             assert.equal(reactHtml, htmlExpected);
         });
+
+        it('should parse br elements without warnings', function() {
+            var htmlInput = '<div><p>Line one<br>Line two<br/>Line three</p></div>';
+            var htmlExpected = '<div><p>Line one<br/>Line two<br/>Line three</p></div>';
+
+            var reactComponent = parser.parse(htmlInput);
```

Comment:
I also think you should've kept the assertion that React children aren't created, as it is extremely unclear from the test what is asserted now and how (the assertion is all implicit).